packages/autocomplete-core/src/__tests__/createAutocomplete.test.ts

```diff
@@ -197,7 +197,7 @@ describe('createAutocomplete', () => {
               createMultiSearchResponse<{ label: string }>(
                 ...requests.map(({ indexName, query = '' }, index) => ({
                   hits: Array.from({ length: 2 }).map((_, i, arr) => ({
-                    objectID: String(index * arr.length + i + 1),
+                    objectID: `${String(index * arr.length + i + 1)}-${query}`,
```

Comment:
No events were sent in response to typing "a" before this, maybe due to a caching mechanism?

Reply:
That's because `openOnFocus=true`, so the event was probably sent as the search was focused and since the results didn't change with the "a" query, we don't send them another time.